boot/bootdata/hivedef.inf

```diff
@@ -2046,14 +2046,14 @@ HKCU,"AppEvents\Schemes\Apps\.Default\SystemAsterisk",,0x00000012
 HKCU,"AppEvents\Schemes\Apps\.Default\SystemAsterisk\.Current","",0x00020000,""
 HKCU,"AppEvents\Schemes\Apps\.Default\SystemAsterisk\.Default","",0x00020000,""
 HKCU,"AppEvents\Schemes\Apps\.Default\SystemExclamation",,0x00000012
-HKCU,"AppEvents\Schemes\Apps\.Default\SystemExclamation\.Current","",0x00020000,""
-HKCU,"AppEvents\Schemes\Apps\.Default\SystemExclamation\.Default","",0x00020000,""
+HKCU,"AppEvents\Schemes\Apps\.Default\SystemExclamation\.Current","",0x00020000,"%SystemRoot%\media\ReactOS_Notice.wav"
```

Comment:
Please revert this and other registry entries. We don't always need to set them. They are automatically set at 2nd setup stage for Workstation configuration if the appropriate sound file exists. But for Server, we keep the default behaviour, without sounds enabled, excepting logon sound. Same as in Windows Server 2003 SP2. See the commit: cbf8b367f5873f8ed9bc558326e8270717d58f45.